examples/cpp/BUILDING.md

```diff
@@ -0,0 +1,115 @@
+# Building
+
+## Prerequisites (All Platforms)
+* [CMake 3.8](https://cmake.org/) or newer
+* C++11 compiler *(see platform specific requirements)*
+* [Doxygen 8.11](http://www.stack.nl/~dimitri/doxygen/download.html) *(for building source documentation)*
```

Comment:
You lifted the BUILDING.md from the project and didn't clean it up. This is not a dependency of the examples. Many of the other items in here are not relevant to the examples.

Reply:
Cleaned out Doxygen references. General cleanup.